docs-src/clusters/quick-install.md

```diff
@@ -1,85 +1,47 @@
 ---
 id: quick-install
-title: How to quickly install a Temporal Cluster for testing and local development
+title: How to install a Temporal Cluster quickly for testing and local development
 sidebar_label: Run a development Cluster
 description: There are four ways to quickly install and run a Temporal Cluster.
 ---
 
-The following sections list various methods of deploying your [Temporal Clusters](/concepts/what-is-a-temporal-cluster) locally, so that you can use and interact with the [Temporal Client](/concepts/what-is-a-temporal-client) APIs and [tctl](/tctl-v1) commands to test and develop applications.
-
-The following sections list methods for deploying your Temporal development Clusters.
-
-- [Temporalite](#temporalite): This distribution of Temporal runs as a single process with zero runtime dependencies.
-- [Docker](#docker-compose): Using Docker Compose simplifies developing your Temporal Application.
-- [Gitpod](#gitpod): One-click deployments are available for Go and TypeScript.
+The following section describes how to deploy your [Temporal Clusters](/concepts/what-is-a-temporal-cluster) locally so that you can use and interact with the [Temporal Client](/concepts/what-is-a-temporal-client) APIs and [tctl](/tctl-v1) commands to test and develop applications.
 
 **For information on deploying a [production environment](/server/production-deployment), see the [Temporal Cloud](/cloud) documentation.**
 
-### Temporalite
+### Temporal CLI
 
-Temporalite is a distribution of Temporal that runs as a single process with zero runtime dependencies.
+Temporal CLI is a distribution of Temporal that runs as a single process with zero runtime dependencies.
 It supports persistence to disk and in-memory mode through SQLite.
 
-**Prerequisites**
-
-Temporalite requires [Go 1.18 or later](https://go.dev/dl/).
-
-**Build and start Temporalite**
-
-The following steps start and run a Temporal Cluster.
-
-1. Build from source.
-   ```bash
-   git clone https://github.com/temporalio/temporalite.git
-   cd temporalite
-   go build ./cmd/temporalite
-   ```
-2. Start Temporalite by using the `start` command.
-   ```bash
-   ./temporalite start --namespace default
-   ```
-   Replace `default` with your [Namespace Name](/namespaces).
-
-**Results**: You should have Temporal Cluster running at `http://127.0.0.1:7233` and the Temporal Web UI at [`http://127.0.0.1:8233`](http://127.0.0.1:8233/).
+**Install and start the Temporal CLI**
 
-<!-- For macOS users, if you receive the `error setting up schema: stat /Users/<user_name>/Library/Application Support/temporalite/db:` error, then create the folders `temporalite/db` in your `Application Support` library. -->
+1. Choose one of the following install methods.
```

Comment:
Can we have a tab per OS?

Mac options:
1. brew install temporal
2. curl | sh
3. download a release, extract and put in PATH

Linux:
2 and 3

Windows:
2 and 3

Reply:
Bonus points if we can make the tab for your OS active automatically.